tools/internal_ci/linux/grpc_distribtests_python.cfg

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Config file for the internal CI (in protobuf text format)
+
+# Location of the continuous shell script in repository.
+build_file: "grpc/tools/internal_ci/linux/grpc_distribtests_python.sh"
+timeout_mins: 360
```

Comment:
Do we really need 6 hours for this test?

Reply:
Actually... is 6 hours enough? If we are compiling ARM wheels with QEMU in this job, the total time consumption will be at least 7 hours.

Here is a timeout Python artifact ARM build (>7 hours): https://fusion.corp.google.com/runanalysis/info/prod:grpc%2Fcore%2Fmaster%2Flinux%2Fgrpc_build_artifacts_extra/prod:grpc%2Fcore%2Fmaster%2Flinux%2Fgrpc_build_artifacts_extra/KOKORO/7702989d-368e-4f31-9078-2125869f70f3/1615801227889/build%20%237985/Targets

Reply:
The last 2 runs I made (by "kidnapping" and existing PR test job) took between 2 and 2.5 hours. While that's not great, it's also not terrible and it's much faster than the full multilanguage artifacts - packages - distribtest flow.

I can reduce the timeout to 240mins to still allow some buffer.

On ARM7 build: we actually no longer run these, in one of my relatively recent PRs I changed the "linux_extra" so that they are crosscompiled (instead of building under an emulator), which is almost as fast as a regular build.

Reply:
Adjusted to 240mins.